packages/web-components/fast-foundation/src/skeleton/skeleton.template.ts

```diff
@@ -0,0 +1,18 @@
+import { html } from "@microsoft/fast-element";
+import { Skeleton } from "./skeleton";
+
+export const SkeletonTemplate = html<Skeleton>`
+  <template
+    shape="${x => x.shape}"
```

Comment:
You should be able to remove the `shape` binding since `template` is just a stand-in for the element host itself in this case. Since the element has the shape attr, it doesn't need the binding.